### DISCUSSION TITLE
Adding TF to mods

### DISCUSSION THREAD

**Discussion by judg393** [scfmod/FS22_TerraFarm#109] [Q&A] [answered]
So first yay updates to TF 

2ndly I guess adding mod support is different now? 

I worked out how to add it using the old TF but it doesn't seem to work in the new version how can I do this :)

**Comment by scfmod** [comment, marked as answer]
Yeah it's a lot easier to add configurations now :-)
You can add it
- Directly to the vehicle XML using specialization
- Using "machineConfigurations.xml" inside your mod
- Using a separate configuration mod like [the official addon mod](https://github.com/scfmod/FS22_TerraFarmMachines)

[Documentation](https://github.com/scfmod/FS22_TerraFarm/blob/main/docs/INDEX.md)

If you want me to add your configurations to the official addon mod [create a new issue](https://github.com/scfmod/FS22_TerraFarmMachines/issues) with config files and I'll add them :-)

**Reply by judg393** [reply]
Thank you I discovered the Official addon had the mod bucket i was trying to add anyway will have to play a bit with that to see how it all works :D
